integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -752,6 +753,20 @@ public static void verifyIntrospectorRuns(String domainUid, String domainNamespa
     logger.info("Verifying introspector pod is created, runs and deleted");
     String introspectJobName = getIntrospectJobName(domainUid);
     checkPodExists(introspectJobName, domainUid, domainNamespace);
+
+    String labelSelector = String.format("weblogic.domainUID in (%s)", domainUid);
+    V1Pod introspectorPod = assertDoesNotThrow(() -> getPod(domainNamespace, labelSelector, introspectJobName),
+        "Could not get introspector pod");
+    assertTrue(introspectorPod != null && introspectorPod.getMetadata() != null,
+        "introspector pod or metadata is null");
+    try {
+      String introspectorLog = getPodLog(introspectorPod.getMetadata().getName(), domainNamespace);
+      logger.info("Introspector pod log START***********************************");
```

Comment:
Can you remove the asterisk(s). from the logger command. 
logger.info("Introspector pod log START");
logger.info("Introspector pod log  END");

Reply:
done